contrib/contribution-guidelines.md

```diff
@@ -1,3 +1,39 @@
 # Elite Dangerous Pilot's Network - Contribution Guidelines
 
-{{ DRAFT PLACEHOLDER - This document will hold all of the expectations of each Community Member's contributions to the Project. }}
+## What do I need to know to help?
+As our organization spans multiple subprojects, each one uses its own unique technology stack, including various programming languages, frameworks, or tools. Familiarize yourself with the specific technologies used in the subproject you're interested in by checking the README file or the project's documentation.
+
+## How do I make a contribution?
+Here's a detailed overview of how to contribute code or documentation:
+
+1. Find (or create) an issue that you are interested in addressing or a feature that you would like to add in the subproject you're focusing on. Refer to the [guidelines](#issue-creation-and-linking) for more information and check the GitHub issues page of the relevant project.
+2. Fork the repository associated with the issue to your local GitHub organization. This means that you will have a copy of the repository under `your-GitHub-username/repository-name`.
+3. Clone the repository to your local machine using `git clone https://github.com/github-username/repository-name.git`.
+4. Create a new branch for your fix using `git checkout -b {issue-number}-branch-name-here`, make sure to include the issue number in your branch name.
+5. Make the appropriate changes for the issue you are trying to address or the feature that you want to add. Make sure to comply with our git commit best practices, listed [here](https://github.com/trein/dev-best-practices/wiki/Git-Commit-Best-Practices).
+6. Use `git add insert-paths-of-changed-files-here` to add the file contents of the changed files to the "snapshot" git uses to manage the state of the project, also known as the index.
+7. Use `git commit -m "{issue-numer} Insert a short message of the changes made here"` to store the contents of the index with a descriptive message. Make sure to include the issue number as the first part of the message. This way we can more easily track which changes were made in function of which ticket.
+8. Push the changes to the remote repository using `git push origin issue-number}-branch-name-here`.
+9. Submit a pull request to the upstream repository.
+10. Title the pull request with a short description of the changes made and the issue or bug number associated with your change.
+11. In the description of the pull request, explain the changes that you made, any issues you think exist with the pull request you made, and any questions you have for the maintainer. It's OK if your pull request is not perfect (no pull request is), the reviewer will be able to help you fix any problems and improve it!
+12. The pull request needs to be approved by at least two people, one of whom needs to be the relevant domain's workgroup lead or part of the steering committee.
```

Comment:
Update: 

'workgroup lead' to 'Work Group Lead'

(capitalize) 'Steering Committee'